public/app/plugins/panel/geomap/editor/MapZoomEditor.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import { StandardEditorProps } from '@grafana/data';
+import { GeomapPanelOptions } from '../types';
+import { NumberInput } from '../components/NumberInput';
+
+export const MapZoomEditor: FC<StandardEditorProps<number | undefined, any, GeomapPanelOptions>> = ({
+  value,
+  onChange,
+  context,
+}) => {
+  // TODO:
+  // Somehow use context to get the current map and listen to zoom changes
```

Comment:
Why not hooking up the panel.onPanelOptionsChange callback to zoom interaction directly in the panel implementation? I don't think options pane should ever reference the implemetation detail of the panel

Reply:
because it should be explicitly set from the editor -- In the editor I want two things:
* show your current zoom/view
* press a button to use it

We can (and likely should) send the current view over the EventBus -- this is fine for getting changes, but you can not get the current value :(